components/pedigree/resources/src/main/resources/pedigree/pedigree.js

```diff
@@ -179,6 +183,8 @@ define([
                     this._partnershipMenu = this.generatePartnershipMenu();
                     this._exportSelector = new ExportSelector();
                     this._printDialog = new PrintDialog();
+                    var studies = editor.getPreferencesManager().getConfigurationOption("studies");
```

Comment:
the list of studies should be obtained inside `StudySelectionDialog` constructor, no need to clutter the main file (since both places have access to `editor.getPreferencesManager().getConfigurationOption("studies")`